spec/lib/http/form_data/multipart_spec.rb

```diff
@@ -95,4 +95,13 @@ def disposition(params)
     subject { form_data.content_length }
     it { is_expected.to eq form_data.to_s.bytesize }
   end
+
+  describe "#boundary" do
+    subject { form_data.boundary }
+    it { is_expected.not_to be_empty }
+
+    it "is included in content type" do
+      expect(form_data.content_type).to end_with(form_data.boundary)
```

Comment:
@ixti Make sense, I noticed now the `let(:boundary)` regex that I can use instead of the `not_to be_empty` assertion.